common/components/urls/urls_v2.json

```
@@ -6,6 +6,7 @@
   {"name": "CreateEventProject", "pattern": "events/(?P<event_id>[\\w\\-]+)/projects/create/(?P<project_id>[\\w\\-]*)"},
   {"name": "AboutEventProject", "pattern": "events/(?P<event_id>[\\w\\-]+)/projects/(?P<project_id>[\\w\\-]+)"},
   {"name": "CreateProject", "pattern": "projects/create/(?P<id>[\\w\\-]*)"},
+  {"name": "IframeProject", "pattern": "ips/(?P<id>[\\w\\-]+)"}, 
```

My preference would be for urls of iframe versions of pages to branch off from the main urls.
```suggestion
  {"name": "IframeProject", "pattern": "projects/ips/(?P<id>[\\w\\-]+)"}, 
```

---

done but its projects/inframe/ and groups/inframe/